server/views/pages/overview/partials/overviewTabActionPlanColumn.njk

```diff
@@ -0,0 +1,101 @@
+{% if not actionPlan.problemRetrievingData %}
+
+  <div class="moj-page-header-actions">
+
+    <div class="moj-page-header-actions__title">
+      <h2 class="govuk-heading-m">Goals in progress</h2>
+    </div>
+
+    {% if hasEditAuthority %}
+
+      <div class="moj-page-header-actions__actions">
+        <div class="moj-button-menu">
+          <div class="moj-button-menu__wrapper">
+
+            {{ govukButton({
+              text: "Add a new goal",
+              href: "/plan/" + prisonNumber + "/goals/create",
+              id: "add-goal-button",
+              classes: "moj-button-menu__item moj-page-header-actions__action"
+            }) }}
+
+          </div>
+        </div>
+      </div>
+
+    {% endif %}
+
+  </div>
+
+  {% for goal in actionPlan.goals %}
+    <div class="govuk-summary-card">
+      <div class="govuk-summary-card__title-wrapper">
+        <h2 class="govuk-summary-card__title">Goal {{ loop.index }}</h2>
+        <ul class="govuk-summary-card__actions">
```

Comment:
Here are the new actions/controls for the Goal summary cards - I copied this from the prototype; I hope its correct @chrimesdev  ??

The Update link is controlled by the feature toggle

I've added the Archive link, but commented it out - I don't believe its MVP, but as and when we get to do that story(ies) the link is here for us 👍